### PR TITLE
Add back mint fee getter

### PR DESCRIPTION
**Pull request by oveddan**

### CODE REVIEW

**Comment by oveddan** [comment]
Current dependencies on/for this PR:
* main
  * **PR #193** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/193" target="_blank">Graphite</a> 
    * **PR #184** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/184" target="_blank">Graphite</a> 
      * **PR #161** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/161" target="_blank">Graphite</a> 
        * **PR #198** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/198" target="_blank">Graphite</a>  👈
        * **PR #189** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/189" target="_blank">Graphite</a> 
          * **PR #190** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/190" target="_blank">Graphite</a> 
            * **PR #191** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/191" target="_blank">Graphite</a> 
            * **PR #188** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/188" target="_blank">Graphite</a> 
            * **PR #187** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/187" target="_blank">Graphite</a> 
        * **PR #179** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/179" target="_blank">Graphite</a> 
          * **PR #171** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/171" target="_blank">Graphite</a> 
            * **PR #183** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/183" target="_blank">Graphite</a> 
            * **PR #178** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/178" target="_blank">Graphite</a> 
            * **PR #175** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/175" target="_blank">Graphite</a> 
            * **PR #173** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/173" target="_blank">Graphite</a> 
            * **PR #172** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/172" target="_blank">Graphite</a> 
        * **PR #170** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/170" target="_blank">Graphite</a> 
        * **PR #168** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/168" target="_blank">Graphite</a> 
          * **PR #169** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/169" target="_blank">Graphite</a> 
        * **PR #166** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/166" target="_blank">Graphite</a> 
        * **PR #164** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/164" target="_blank">Graphite</a> 
        * **PR #163** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/163" target="_blank">Graphite</a> 
        * **PR #162** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/162" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/198?utm_source=stack-comment).

File: src/nft/ZoraCreator1155Impl.sol [hunk 1]
```diff
@@ -482,6 +482,10 @@ contract ZoraCreator1155Impl is
         return firstMinters[tokenId];
     }
 
+    function mintFee() external view returns (uint256) {
```
Comment (iainnash) [inline src/nft/ZoraCreator1155Impl.sol:485]: Is/was this in the interface?

**Review by iainnash — APPROVED** [review]

File: src/nft/ZoraCreator1155Impl.sol (hunk 1 above)
Comment (iainnash) [reply src/nft/ZoraCreator1155Impl.sol:485]: If so, should add it back in. If not, as-is is fine.

Comment (oveddan) [reply src/nft/ZoraCreator1155Impl.sol:485]: it looks like it was in the [IMintFeeManager]([url](https://github.com/ourzora/zora-1155-contracts/blob/main/src/interfaces/IMintFeeManager.sol#L9)) interface, which this no longer inherits from; should we still add it here?

Comment (oveddan) [reply src/nft/ZoraCreator1155Impl.sol:485]: added in the most recent commit

Comment (iainnash) [reply src/nft/ZoraCreator1155Impl.sol:485]: @oveddan does it affect the 165 of the entire contract? i think a test should catch this.

**Pull request merged**